assets/company-admin/components/CompanyAdminApp.tsx

```diff
@@ -69,7 +69,7 @@ interface IDispatchProps {
     setProductFilter: (productId: any) => any;
 }
 
-interface IOwnProps {
+type IOwnProps = {
```

Comment:
why defining it when it's empty?

Reply:
because I want to see what type I pass to the `connect` function. for the most part documentation

Reply:
I see, didn't notice you use it there, if you would just pass there `{}` wouldn't it be enough?

Reply:
It would, but I wanted to have a clear link between connect and the component props. And in the future if we extend this component and decide to pass some props it will be clear and easier to do it.

Reply:
Technically it would but then it's not obvious what that parameter means. This way we have a direct link between props and that type parameter.